Orange/widgets/evaluate/owpredictions.py

```diff
@@ -682,10 +801,18 @@ def merge_data_with_predictions():
 
         if self.data:
             text = self._get_details().replace('\n', '<br>')
-            if self.selected_classes:
-                text += '<br>Showing probabilities for: '
-                text += ', '. join([self.class_values[i]
-                                    for i in self.selected_classes])
+            if self.is_discrete_class and self.shown_probs != self.NO_PROBS:
+                text += '<br>Showing probabilities for '
+                if self.shown_probs == self.MODEL_PROBS:
+                    text += "all classes known to the model"
+                elif self.shown_probs == self.DATA_PROBS:
+                    text += "all classes that appear in the data"
+                elif self.shown_probs == self.BOTH_PROBS:
+                    text += "all classes that appear in the data " \
+                            "and are known to the model"
+                else:
+                    class_idx = self.shown_probs - len(self.PROB_OPTS)
+                    text += f"'{self.class_var.values[class_idx]}'"
```

Comment:
The `target_class` value is missing in the report.

Reply:
Are you sure you want to display it for regression as well?

Reply:
Sure not.